### PR TITLE
Add --version command line option

### PR DESCRIPTION
**Pull request by cjmayo**
Split from #3242.

Copyright and mention of Friends-of-Fritzing e.V. need review.

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -194,7 +197,23 @@ int main(int argc, char *argv[])
...
 				"The process path follows the -ep argument; the name of the menu item follows the -epname argument;\n"
 				"and any arguments to pass to the external process are provided in the -eparg arguments.\n";
+            break;
 		}
+        case FInitResultVersion:
+        {
+            QTextStream cout(stdout);
+            cout <<
+                "Fritzing " << Version::versionString() << "\n"
+                "Copyright (C) 2016 ???.\n"
```
Comment (ovidiub13) [inline src/main.cpp]: You can add `2019`. We plan several releases this year.

File: src/main.cpp [hunk 2]
```diff
@@ -194,7 +197,23 @@ int main(int argc, char *argv[])
...
+                "Source code License GPLv3: GNU GPL version 3 <https://www.gnu.org/licenses/gpl.html>.\n"
+                "Documentation and Part Designs License CC BY-SA 3.0: Creative Commons Attribution-ShareAlike 3.0 Unported <https://creativecommons.org/licenses/by-sa/3.0/legalcode>.\n"
+                "This is free software: you are free to change and redistribute it.\n"
+                "There is NO WARRANTY, to the extent permitted by law.\n"
+                "\n"
+                "Maintained by the Friends-of-Fritzing e.V.\n";
```
Comment (ovidiub13) [inline src/main.cpp]: @KjellMorgenstern is this right?

File: src/main.cpp [hunk 3]
```diff
@@ -194,7 +197,23 @@ int main(int argc, char *argv[])
...
+            break;
+        }
+        }
```
Comment (ovidiub13) [inline src/main.cpp]: Can you format the indentation?

**Review by madpipeline — APPROVED** [review]

File: src/main.cpp (hunk 2 above)
Comment (ovidiub13) [reply src/main.cpp]: After some discussion with @KjellMorgenstern, we've think it's best for now to not have the © in the `--version` message due to the complicated legal situation of Fritzing right now.
In order not to hold this PR back: can you strip this message to just the version print? We'll revisit the © part later.

**Review by madpipeline — CHANGES_REQUESTED** [review]

**Comment by cjmayo** [comment]
Stripped back to version number only.
Replaced all tabs with spaces to improve the indentation.

**Review by KjellMorgenstern — APPROVED** [review]

**Pull request merged**